libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/LocationOverlay.kt

```diff
@@ -90,20 +90,23 @@ class LocationOverlay(private val mapView: CycleMapView) :
     }
 
     fun lockOnLocation() {
-        lockedOn = true
```

Comment:
as far as I can tell, `lockedOn` is now an unused variable, can we remove it please?

Reply:
Yes, you're right. It's exactly this kind of mistake I was trying to avoid by getting some sleep ... Should have made a note.